app/workers/evss/disability_compensation_form/upload_bdd_instructions.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module EVSS
+  module DisabilityCompensationForm
+    class UploadBddInstructions < Job
+      FORM_TYPE = '21-526EZ'
+      STATSD_KEY_PREFIX = 'worker.evss.submit_form526_bdd_instructions'
```

Comment:
Are the constants on lines 6 and 7 used at all?

Reply:
STATSD_KEY_PREFIX is referenced in lib/common/client/concerns/monitoring.rb:
   22:           increment("#{self.class::STATSD_KEY_PREFIX}.#{caller}.total")
   30:           increment("#{self.class::STATSD_KEY_PREFIX}.#{caller}.fail", tags)